src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateClaimFormCallbackHandler.java

```diff
@@ -62,7 +75,42 @@ private CallbackResponse generateClaimForm(CallbackParams callbackParams) {
             callbackParams.getParams().get(BEARER_TOKEN).toString()
         );
 
-        caseDataBuilder.systemGeneratedCaseDocuments(wrapElements(sealedClaim));
+        if (stitchEnabled && (caseData.getRespondent1Represented().equals(YesOrNo.NO)
+            || ofNullable(caseData.getRespondent2Represented()).isPresent()
+            && caseData.getRespondent2Represented().equals(YesOrNo.NO))) {
```

Comment:
if (stitchEnabled && ((YesOrNo.NO).equals(caseData.getRespondent1Represented())
            || (YesOrNo.NO).equals(caseData.getRespondent2Represented()) ) {

if you put the yes/no part first and check against that you won't need the of nullable part?

Reply:
swapped out :)